src/components/CourseForm.jsx

```diff
@@ -1,77 +1,96 @@
-import { React, useState} from 'react';
-import { Button, Modal, Form} from 'react-bootstrap';
+/* eslint-disable valid-jsdoc */
```

Comment:
requiring `valid-jsdoc` is overkill IMO. You can disable this rule for the entire project by adding it to rules in `.eslintrc.json`:

```
   "rules": {
        "require-jsdoc": 0
    }
    ```